src/coreclr/src/gc/gcpriv.h

```diff
@@ -2873,6 +2871,8 @@ class gc_heap
     size_t get_current_allocated();
     PER_HEAP_ISOLATED
     size_t get_total_allocated();
+    PER_HEAP_ISOLATED
+    size_t get_total_gen_fragmentation (int gen_number);
```

Comment:
>size_t get_total_gen_fragmentation (int gen_number); [](start = 4, length = 52)

I don't understand why this is moved?

Reply:
It was a protected method.

Reply:
there's no access level change between the original place and the new place. plus GCHeap can access protected members of gc_heap. 

---
In reply to: [381707206](https://github.com/dotnet/runtime/pull/32580#discussion_r381707206) [](ancestors = 381707206)